android/src/fdroid/play/listings/hu-HU/full-description.txt

```diff
@@ -0,0 +1,52 @@
+‣ Ingyenes alkalmazásunk nem követi Önt, nem tartalmaz hirdetéseket, és az Ön adományaiból működik.
+‣ Kis csapatunkkal folyamatosan fejlesztjük szabadidőnkben, saját pénzünkből és felhasználóink adományaiból/hozzájárulásaiból.
+‣ Ha valami hibás vagy hiányzik a térképen, kérjük, javítsdaki az <b>OpenStreetMap</b> oldalon, és a jövőbeni térképfrissítésekben láthatja a változtatásokat.
+‣ Ha a navigáció vagy a keresés nem működik, kérjük, írjon nekünk e-mailt. <i>MINDEN</i> e-mailre válaszolunk, és azonnal a hibát kijavítjuk!
+
+<b>A legjobb motivációnk a visszajelzések és az 5 csillagos értékelések!</b>
+
+Főbb jellemzők:
+- Ingyenes, nyílt forráskódú, nincs reklám, nincs nyomon követés.
+- Részletes offline térképek olyan helyekkel, amelyek a Google térképeken nem is léteznek – hála az <b>OpenStreetMap</b> közösségnek.
+- Kerékpáros útvonalak, túraútvonalak és gyalogos ösvények.
+- Szintvonalak, magassági profilok, csúcsok és lejtők.
+- Részletes gyalogos, kerékpáros és KÍSÉRLETI autós navigáció hangutasítással
+- Gyors offline keresés
+- Könyvjelzők exportálása és importálása KML/KMZ formátumban (hamarosan GPX formátumban is).
+- Sötét üzemmód a szem védelméért.
+
+Az Organic Maps-ben <i>még</i> nincs Android Auto, tömegközlekedés, műholdas térképek és egyéb menő funkciók. De az Ön <i>segítségével és támogatásával</i>, lépésről lépésre jobbá tehetjük a világot.
+
+Az Organic Maps <b>tiszta és organikus, szeretettel készült</b>:
+
+- Villámgyors offline élmény
+- Tiszteletben tartja a magánéletet
+- Kíméli az akkumulátort
+- Nincsenek váratlan mobil adatforgalmi díjak
+- Egyszerűen használható, csak a legfontosabb funkciókat tartalmazza
```

Comment:
Let's use `• ` instead of `- ` everywhere, for consistency with other translations.